src/main/java/com/code_intelligence/jazzer/driver/FuzzTargetRunner.java

```diff
@@ -111,7 +126,7 @@ public final class FuzzTargetRunner {
     try {
       fuzzTarget = FuzzTargetFinder.findFuzzTarget(fuzzTargetClass);
     } catch (IllegalArgumentException e) {
-      err.printf("ERROR: %s%n", e.getMessage());
+      Log.error(e);
```

Comment:
This will not print the error message anymore.

Reply:
It would, but it would also emit the stack trace. Instead, I changed it to again print the message only.